DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -194,3 +194,72 @@ ul.no-bullets {
 .vertical-align-centre {
     vertical-align: middle
 }
+
+.side-nav-menu {
+  @include govuk-media-query($until: desktop) {
+    display: none;
+  }
+}
+
+.side-nav-menu-bottom {
+  @include nhsuk-responsive-padding(0, "left");
+
+  @include govuk-media-query($from: desktop) {
+    display: none;
+  }
+}
+
+.side-nav__heading {
+  font-weight: 600;
+  color: $color_nhsuk-grey-1;
+  @include nhsuk-responsive-margin(3, "bottom");
+  @include nhsuk-responsive-padding(1, "top");
+  font-size: 1.1875rem;
+  line-height: 1.25;

```

Comment:
Re font-styles -- should we use any of the Typography classes for that rather than setting the font-size, font-weight and line-height explicitly?
https://service-manual.nhs.uk/design-system/styles/typography